src/backend/Post/BookController.js

```diff
@@ -11,6 +11,8 @@ exports.getBooks = (req, res) => {
 
   bookQuery
     .find({})
+    .sort(req.query.title)
```

Comment:
What happens if both query params are missing from the request?

Reply:
If both of the query params are omitted or empty, result on back end is by chronological order from books collections. (tested in postman).
On client side, initially both params are empty, books are shown in chronological order.